packages/roosterjs-editor-types/lib/interface/TableFormat.ts

```diff
@@ -7,45 +8,45 @@ export default interface TableFormat {
     /**
      * Top border color for each row
      */
-    topBorderColor?: string | null;
+    topBorderColor: string | null;

```

Comment:
Why need to change this file?
The template class `Required<T>` can make everything required, no need to change this file